packages/backend/src/core/activity/counters/RpcCounter.ts

```diff
@@ -65,7 +62,9 @@ export function createRpcCounter(
           }
         })
 
-        const blocks = await promiseAllPlus(queries, logger)
+        const blocks = await promiseAllPlus(queries, logger, {
+          taskQueueId: `${projectId.toString()}-${createRpcCounter.name}`,
```

Comment:
```suggestion
          taskQueueId: `${projectId.toString()}-activityRpcCounter`,
```
But anyway, I don't think we need it that much. We have metrics for activity in the `SequenceProcessor`